### PR TITLE
Like functionality

### PR DESCRIPTION
**Pull request by StefanPahlplatz**

### CODE REVIEW

File: src/modules/feed/components/Transaction/Header.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { Dropdown } from "semantic-ui-react";
 import moment from "moment-twitter";
 import { route } from "preact-router";
 
-import { PATH_ADD_TRANSACTION } from "../../../../../routes";
+import { PATH_ADD_TRANSACTION } from "../../../../routes";
```
Comment (matthijsgroen) [inline src/modules/feed/components/Transaction/Header.js]: For readability, would it help to link to it from the root? (using a webpack resolve alias?)

File: src/modules/feed/components/Transaction/LikeButton.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    update={(cache, { data: { toggleLikePost } }) =>
+      updateState(cache, toggleLikePost)
+    }
+  >
+    {mutate => (
+      <Button
+        size="mini"
+        basic
+        className="button-action"
+        onClick={() =>
```
Comment (matthijsgroen) [inline src/modules/feed/components/Transaction/LikeButton.js]: Maybe extract this click handler into seperate function?

File: src/modules/feed/components/Transaction/LikeButton.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+              },
+            },
+            update: (proxy, { data: { toggleLikePost } }) =>
+              updateState(proxy, toggleLikePost),
+          })
+        }
+      >
+        <Icon
+          name={liked ? "heart" : "heart outline"}
+          color={liked ? "red" : null}
```
Comment (matthijsgroen) [inline src/modules/feed/components/Transaction/LikeButton.js]:
```suggestion
          color={liked && "red"}
```

File: src/modules/feed/components/Transaction/LikeButton.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import { h } from "preact";
+import { Button, Icon } from "semantic-ui-react";
+import gql from "graphql-tag";
+import { Mutation } from "react-apollo";
+
+import settings from "../../../../config/settings";
```
Comment (matthijsgroen) [inline src/modules/feed/components/Transaction/LikeButton.js]: For readability, would it help to link to it from the root? (using a webpack resolve alias?)

File: src/modules/feed/components/Transaction/Transaction.js [hunk 5]
```diff
@@ -1,7 +1,10 @@
 import { h } from "preact";
 import { Card } from "semantic-ui-react";
-import { LikeButton } from "./LikeButton";
-import Header from "./Header/Header";
+import LikeButton from "./LikeButton";
+import Header from "./Header";
+import settings from "../../../../config/settings";
```
Comment (matthijsgroen) [inline src/modules/feed/components/Transaction/Transaction.js]: For readability, would it help to link to it from the root? (using a webpack resolve alias?)

File: src/support/auth.js [hunk 6]
```diff
@@ -1,15 +1,25 @@
+import client from "../apollo";
 import { route } from "preact-router";
 
 import settings from "../config/settings";
 import { PATH_LOGIN, PATH_CHOOSE_TEAM } from "../routes";
 
+/**
```
Comment (matthijsgroen) [inline src/support/auth.js]: Ok does this reaaaaally add value?

File: src/modules/feed/components/Transaction/LikeButton.js (hunk 3 above)
Comment (StefanPahlplatz) [reply src/modules/feed/components/Transaction/LikeButton.js]: This gives me the error: 
```
Warning: Failed prop type: Invalid prop `color` of value `false` supplied to `Icon`, expected one of 
["red","orange","yellow","olive","green","teal","blue","violet","purple","pink","brown","grey","black"].
```

**Pull request merged**